test/redirects/redirects.csv

```diff
@@ -8,8 +7,8 @@
 /dora-joins-google-cloud.html,/news/dora-joins-google-cloud/,301
 /2018/12/dora-joins-google-cloud/,/news/dora-joins-google-cloud/,301
 /core,/research/,301
-/concepts/wellbeing,/devops-capabilities/cultural/well-being/,302
-/devops-capabilities/cultural/wellbeing,/devops-capabilities/cultural/well-being/,302
+/concepts/wellbeing,/capabilities/well-being/,302
+/capabilities/wellbeing,/capabilities/well-being/,302
```

Comment:
should be 
`/devops-capabilities/cultural/wellbeing,/capabilities/well-being/,302`